.vscode/launch.json

```
@@ -40,6 +40,27 @@
             "purpose": ["debug-test"],
             "console": "integratedTerminal",
             "justMyCode": false
+          },
+          {
+            "name": "Debug Evaluation",
+            "type": "debugpy",
+            "request": "launch",
+            "cwd": "${workspaceFolder}/evaluation",
+            "module": "scripts",
+            "args": [
+                "evaluate",
+                "--config=./example_config.json",
```

This looks stale, perhaps:
```suggestion
                "--config=./config.json",
```